lib/apis/v3/felixconfig.go

```diff
@@ -246,6 +246,9 @@ type FelixConfigurationSpec struct {
 	// modes can use XDP. This is not recommended since it doesn't provide better performance than
 	// iptables. [Default: false]
 	GenericXDPEnabled *bool `json:"genericXDPEnabled,omitempty" confignamev1:"GenericXDPEnabled"`
+
+	// IPtablesBackend
+	IptablesBackend string `json:"iptablesBackend,omitempty" validate:"omitempty,iptablesBackend"`
```

Comment:
I believe it should be `IPTablesBackend`

Reply:
I'm fine with that but I feel like the command is always `iptables`, no capitalization, so I thought just the 1st I should be capitalized (even though I was inconsistent with that).

Reply:
Yeah, I can see that. I guess it doesn't matter much.